test/utils/server.js

```diff
@@ -61,11 +61,11 @@ function bundleTests(name) {
 
       res.set('Content-Type', 'application/javascript');
 
-      var b = browserify(testFiles[name]);
-      b.transform(aliasify);
-      var str = b.bundle({
+      var b = browserify(testFiles[name], {
         insertGlobals: true
       });
+      b.transform(aliasify);
+      var str = b.bundle();
```

Comment:
there's been an api change to browserify wherein options must be passed not into `bundle` but now in `browserify`'s constructor.